server.js

```diff
@@ -1,9 +1,17 @@
 const express=require("express");
+const mongoose= require("mongoose");
 const app=express();
 const port=3000
 
+const data='mongodb+srv://aniketg:FOT8yjNAFZj39q0m@cluster0.wikddym.mongodb.net/?retryWrites=true&w=majority'
+
 app.get("/",(req,res)=>{
-    res.send("Weirdy Foody");
+    mongoose.connect(data)
+    .then(()=>{
+        res.json({message:"server is running"})})
+    .catch((error)=>{
+        res.json({message:error,message:"server is not running"})

```

Comment:
Avoid sending the error message directly in the response. This could potentially expose sensitive information. Instead, log the error and send a generic error message to the client. [important]